sdks/explanations/data-model/data-model-patient.md

```diff
@@ -0,0 +1,71 @@
+---
+slug: patient
+tags:
+    - data model
+    - data owner
+    - patient
+---
+# Patient
+
+A Patient is the subject of all the medical processes. They can be invited by Healthcare Professionals, who can also 
+associate Data Samples and Healthcare Elements to them.
+As a [Data Owner](/sdks/glossary#data-owner), a Patient can ask to access those data as well as creating Data Samples and
+Healthcare Elements by themselves.
```

Comment:
This sounds like a Patient can only use iCure by invitation from healthcare professionals
```suggestion
A Patient is the subject of all the medical processes. They can register to iCure autonomously or be invited by Healthcare Professionals.
Healthcare professionals can associate Data Samples and Healthcare Elements to Patients.
Additionally, as [Data Owners](/sdks/glossary#data-owner) Patients can also ask to access this data or create
Data Samples and Healthcare Elements by themselves.
```